src/neo4j-arc/common/types/cypherDataTypes.ts

```diff
@@ -141,8 +141,14 @@ export const isCypherPropertyType = (value: any): value is CypherProperty => {
 }
 
 export const isCypherTemporalType = (
-  anything: object
-): anything is CypherTemporalType =>
-  [isDate, isTime, isDateTime, isLocalTime, isLocalDateTime, isDuration].some(
-    tester => tester(anything)
+  anything: unknown
+): anything is CypherTemporalType => {
+  return (
+    isDate(anything) ||
```

Comment:
The type signatures of these functions changed and got more advance (using generics etc), hence this update